sessions.py

```diff
@@ -149,22 +148,24 @@ def isCurrentTimeInSession(self):
 
 def consoleTrack():
     s = SessionTracker()
-    while True:
+    error = false
+    while not error: #this will continue until student scan event fails AFTER adding a new student, which should never happen.
         try:
-            a = raw_input('#>')
-            a = int(a)
+            inputID = raw_input('#>')
+            inputID = int(inputID)
         except KeyboardInterrupt:
             exit(0)
         except ValueError:
             print "Error: Invalid Input (cannot convert to int)"
-            continue
+            continue #This continue statement causes any input prior to the error to be continue, so if you types 401F22, then a = 401
```

Comment:
Continue statement will break loop immediately and start next iteration of loop (slightly counter-intuitive naming in this particular use case).

Reply:
I believe you are correct, I was mistaken because I believed my ID was something like 401F22, but it actually is something like 000401, and the program removed the leading zeroes.